app/components/search_bar/search_bar.android.js

```diff
@@ -52,6 +53,7 @@ export default class SearchBarAndroid extends PureComponent {
         blurOnSubmit: true,
         placeholder: 'Search',
         showCancelButton: true,
+        showArrow: true,
```

Comment:
<img width="1222" alt="screen shot 2018-03-15 at 9 27 58 am" src="https://user-images.githubusercontent.com/6182543/37466050-34548a20-2833-11e8-802c-3c97dd292d14.png">

* left: `showArraw: false`
* right: `showArrow: true`

This is for the search bar, there was a redundant use of the back arrow.  Even though they serve a different purpose (toolbar arrow, goes back; search bar arrow, hides keyboard).

It's fine to remove the arrow from the search bar since we can leverage the system button to hide the keyboard on Android.